bigml/model.php

```diff
@@ -324,7 +324,7 @@ public function predict_probability($input_data, $by_name=true,
          //                 respectively.  If True, returns a list of probabilities
          //                 ordered by the sorted order of the class names.
 
-       if ($this->regression) {
+       if ($this->regression_ready) {
```

Comment:
I cannot find any other occurrence of regression_ready in this commit -- so it is not clear to me what this change entails...

Reply:
In the Model constructor, there is no variable called 'regression', it uses the name 'regression_ready'. It's a non-obvious name, but this way it is called correctly.